### ISSUE TITLE
Conflict with the Magic Item module

### ISSUE THREAD

**Issue by MisterHims** [sdenec/tidy5e-sheet#259]
Hello,

I recently noticed that the Tidy Sheet don't working properly with the Magic Items module. Indeed, since the last update (but this maybe earlier) the magic items added to the actors appear but are not magic (no magic wand icon, no new spells added).

I didn't particularly pay attention to console errors and the issue doesn't reoccur after disabling/enabling the module. I don't know if that issue would come from Tidy Sheet or rather from Magic Items but in the doubt, I prefer to warn you both.

Oh and wait! I want to thank you for the great job you do with this wonderful character sheet who change the life :D

**Comment by sdenec** [comment]
I'm not sure I fully understand the issue. Does this issue still exist or was ist a one time thing that was fixed by disabling/enabling the module(s)? I just tried to creat a new magic item with some spells on it and when I give it to a character it whows up fine - with icon, and spell (as long as the requirements to activate the item, eg. equipped and attuned if set up this way) is met.
So I can't confirm any errors with the sheet or magic items.

Just to be sure - you are on the most recent version for tidy5e and didn't got stuck on 0.4.0? If you are still on that version please uninstall and reinstall the sheet module.

**Comment by MisterHims** [comment]
Thanks for the answer and sorry for the little confusion.

That issue was temporary, I had wanted to add a magic item to one of my actor but it didn't work correctly despite the item's conditions being met (no icon and the kinds were added). I also tried on several other actors, without success. However, it worked on actors that I had already added a magic item a while ago.

While looking for a solution to this issue, I disabling/enabling Tidy Sheet and the error did not occur again. I just wanted to let you know of this in case it happens again.

I'm currently at the 0.4.14 version (the last one I think). If it happens again I will get back to you but it was really just to let you know :)
